libiocage/lib/Release.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, name=None,
         libiocage.lib.helpers.init_zfs(self, zfs)
         libiocage.lib.helpers.init_host(self, host)
 
+        if not libiocage.lib.helpers.validate_name(name):
+            raise NameError(f"Invalid 'name' for Release: '{name}'")
```

Comment:
Why are we validating RELEASE? They aren't created by a user with a name, they come from the project. We should remove this.